stylesheet.css

```diff
@@ -373,3 +394,493 @@ a {
+    font-size: 40px;
+    color: #9490ff;
+    padding-bottom: 20px;
+  }
+
+  .display-project1,
+  .display-project2,
+  .display-project3,
+  .display-project4 {
+    display: flex;
+    flex-direction: column;
+    flex-wrap: wrap;
+    align-items: center;
+    margin-top: 10px;
+    margin-bottom: 60px;
+  }
+
+  .display-project1 {
+    flex-direction: row;
+  }
+
+  .display-project2 {
+    flex-direction: row-reverse;
+  }
+
+  .display-project3 {
+    flex-direction: row;
+  }
+
+  .display-project4 {
+    flex-direction: row-reverse;
+  }
+
+  .project-flex {
+    display: flex;
+    flex-direction: column;
+    justify-items: center;
+    padding-left: 23px;
+  }
+
+  .flex-item {
+    color: white;
+    align-items: center;
+    width: 100%;
+  }
+
+  .holdtext {
+    width: 100%;
+    color: #dfe1e6;
+    font-size: 20px;
+  }
+
+  .navlang-flex {
+    display: flex;
+    flex-wrap: nowrap;
+    width: 100%;
+  }
+
+  .subflex-item {
+    height: 30px;
+    text-align: center;
+    line-height: 30px;
+  }
+
+  .project-text-nav {
+    display: flex;
+    flex-direction: column;
+    justify-content: start;
+    flex-wrap: wrap;
+    align-items: flex-start;
+    width: 40%;
+    height: 100%;
+    margin-top: 20px;
+  }
+
+  .hh2 {
+    width: 327px;
+    font-size: 30px;
+    font-weight: 500;
+    text-align: left;
+  }
+
+  .placeholder-div {
+    width: 59%;
+    height: 448px;
+    padding-bottom: 10px;
+  }
+
+  .ph-image {
+    width: 95%;
+    height: 450px;
+    padding-left: 10px;
+    padding-right: 10px;
+    display: grid;
+  }
+
+  .navlang-flex li {
+    width: auto;
+    display: inline-block;
+  }
+
+  .seeproject {
+    display: inline-block;
+    color: #edecff;
+    width: 89px;
+    height: 50px;
+    background-color: #7d73fc;
+    margin-top: 80px;
+    line-height: 48px;
+    text-align: center;
+  }
+
+  .seep {
+    display: flex;
+    justify-content: left;
+    height: 10px;
+    width: 100%;
+    padding-bottom: 10px;
+  }
+
+  .contact-section,
+  .foota {
+    display: flex;
+    flex-direction: column;
+    flex-wrap: wrap;
+    align-items: center;
+    width: 80%;
+    margin: auto;
+  }
+
+  .contact-section {
+    margin-top: 80px;
+  }
+
+  .cc-sect {
+    width: 100%;
+  }
+
+  .cc-top {
+    padding-top: 40px;
+  }
+
+  .contact {
+    display: block;
+    width: 60%;
+    color: #9390ff;
+    font-size: 32px;
+    text-align: center;
+    padding-bottom: 30px;
+    margin: auto;
+  }
+
+  .mble {
+    display: none;
+  }
+
+  .text-invite {
+    width: 60%;
+    height: 105px;
+    font-size: 20px;
+    color: #ebebff;
+    text-align: center;
+    margin: auto;
+  }
+
+  .myform {
+    display: flex;
+    flex-direction: column;
+    flex-wrap: wrap;
+    align-items: center;
+    width: 100%;
+  }
+
+  .myform input {
+    width: 60%;
+    height: 48px;
+    margin-bottom: 15px;
+  }
+
+  .myform textarea {
+    width: 60%;
+    height: 148px;
+    margin-bottom: 15px;
+  }
```

Comment:
- [ ] kindly add the paddings for your inputs like shown in Figma 

Solarized dark             |  Solarized Ocean
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/89282221/187971961-cfb5eb95-85be-4404-8227-83c32e4ad6c8.png)  |  ![image](https://user-images.githubusercontent.com/89282221/187972028-fe6f7918-dec9-449b-862b-0d7475c6c267.png)